### DISCUSSION TITLE
PCAP Pending && Incomplete

### DISCUSSION THREAD

**Discussion by mohasmr** [Security-Onion-Solutions/securityonion#12690] [Unsupported Versions]
i have a problem since 5 or 6 days with PCAP interface
even i restart (so-sensoroni-restart) and clean sensor (so-sensor-clean) the logs in /opt/so/log/sensoroni/sensoroni.log showing that he processing pcap but with no results

![images](https://github.com/Security-Onion-Solutions/securityonion/assets/135219197/a19d8254-77ae-4145-a588-de623d3bd5c9)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/135219197/c3ab6ab1-50d8-4d8f-8b4f-c66e0c61d4b8)

**Comment by jertel** [comment]
Please do not post duplicate messages. #12689
